### PR TITLE
Navbar and footer UI review

### PR DESCRIPTION
**Pull request by dev-enforced**
In this PR you can find the code made to develop components of navbar and footer that are common components available across most of the pages.
Here is the list of files added for the same
image

Issue for which this PR was made:#1 
Live preview Link: https://deploy-preview-2--eleganz-store.netlify.app/

### CODE REVIEW

**Comment by dev-enforced** [comment]
#1

File: src/routes/Routes.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import {Mockbee,MockMan} from "../pages"
+const RoutesList=()=>{
+    return(
+        <Routes>
+            <Route path="/mockbee" element={<Mockbee />} />
```
Comment (rushikesh3299) [inline src/routes/Routes.jsx:7]: You can use code formatting tools to take care of indentations.

**Review by rushikesh3299 — APPROVED** [review]
I gave a suggestion. Naming conventions you used are awesome

File: src/routes/Routes.jsx (hunk 1 above)
Comment (dev-enforced) [reply src/routes/Routes.jsx:7]: Yes this formatting issue has got resolved here with the [#4903a4b](https://github.com/dev-enforced/eleganz-store/pull/2/commits/4903a4b7bca62c44add0d88bfa2e1fc0b219fb1c) and [#a91688](https://github.com/dev-enforced/eleganz-store/pull/2/commits/a9168880851bbe469add64cbcfd8e9c3f29dcf4d)

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,127 @@
...
+.btn-outline:hover {
+  color: var(--text-color);
+  background-color: var(--primary-color);
+}
+
+.badge-content {
+  position: absolute;
+  top: 10px;
+  clip-path: polygon(100% 0%, 75% 50%, 100% 100%, 0 100%, 0 50%, 0 0);
+  padding: 0.5rem 2rem 0.5rem 1rem;
```
Comment (SaurabhChawda) [inline src/index.css:103]: Try to use the Shorthand property.

File: src/routes/Routes.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import { Mockbee, MockMan } from "../pages"
+const RoutesList = () => {
+    return (
+        <Routes>
+            <Route path="/mockbee" element={<Mockbee />} />
+            <Route path="/mockman" element={<MockMan />} />
+        </Routes>
+    )
+}
+export { RoutesList }
```
Comment (JantuDeb) [inline src/routes/Routes.jsx]: This is really good that you created a routes component.

File: src/components/Navigation/Navigation.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                    <form className="main-input-container">
+                        <GoSearch className="search-icon" />
+                        <input type="text" placeholder="SEARCH FOR PRODUCTS" className="search-input" />
+                    </form>
+                </div>
+                <div className="main-links gentle-flex-gap flex-center flex-wrap">
+                    <Link to="/mockbee" className="link-none text-link" >
+                        LOGIN
+                    </Link>
+                    <Link to="/mockman" className="badge mx-2">
```
Comment (SaurabhChawda) [inline src/components/Navigation/Navigation.jsx:23]: Use NavLink when you have ClassName.
https://medium.com/how-to-react/add-an-active-classname-to-the-link-using-react-router-b7c350473916#:~:text=Active%20class%20is%20required,active%20className%20to%20the%20link.

**Review by JantuDeb — APPROVED** [review]

**Review by SaurabhChawda — APPROVED** [review]

**Pull request merged**